actions/dispatch-workflow/action.yml

```diff
@@ -0,0 +1,116 @@
+name: Dispatch Workflow
+description: Dispatch a github workflow and return the run id
+
+inputs:
+  repo:
+    description: Owner and repo in the form owner/repo
+    required: true
+  workflow:
+    description: The name of the workflow file
+    required: true
+  ref:
+    description: The ref or branch of the repo to use
+    required: true
+  workflow-dispatch-fields:
+    description: |
+      Fields to pass into the workflow dipatch, example:
+        --field distinct_run_name="${DISTINCT_RUN_NAME}" \
+        --field evm-ref="${EVM_REF}"
+    required: false
+  distinct_run_name:
+    description:
+      The distinct run name added to the workflow, this is required to more
+      accurately find the correct workflow that was started
+    required: true
+  actor:
+    description:
+      The github actor used to dispatch the workflow, required to more
+      accurately find the correct workflow that was started
+    required: true
+  github-token:
+    description: The github token to use
+    required: true
+  retry-delay:
+    description:
+      The amount of time in seconds to sleep between calls to check for workflow
+      to be created
+    required: false
+    default: "10"
+  retry-count:
+    description: The max number of times to wait for the workflow to be created
+    required: false
+    default: "6"
+outputs:
+  run-id:
+    description: The found run-id
+    value: ${{ steps.get-run-id.outputs.run-id }}
+
+runs:
+  using: composite
+  steps:
+    - name: Dispatch Workflow
+      shell: bash
+      id: get-run-id
+      env:
+        GH_TOKEN: ${{ inputs.github-token }}
+        DISTINCT_RUN_NAME: ${{ inputs.distinct_run_name }}
+        GH_REPO: ${{ inputs.repo }}
+        WORKFLOW: ${{ inputs.workflow }}
+        REF: ${{ inputs.ref }}
+        ACTOR: ${{ inputs.actor }}
+        EVM_REF: ${{ inputs.evm-ref }}
+        FIELDS: ${{ inputs.workflow-dispatch-fields }}
+        RETRY_DELAY: ${{ inputs.retry-delay }}
+        RETRY_COUNT: ${{ inputs.retry-count }}
+      run: |
+        current_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+        # run the workflow
+        echo "Dispatching the workflow"
+        gh workflow run "${WORKFLOW}" \
+          --repo ${GH_REPO} \
```

Comment:
we don't need this anymore

Reply:
Removed with commit: https://github.com/smartcontractkit/.github/pull/454/commits/e02e6b59110bbf3ca6c14a23d1fa1641d2b0bde0